Documentation/contributing/development/contributing_guide.rst

```diff
@@ -183,13 +211,64 @@ Getting a pull request merged
 
 #. Owners of the repository will automatically adjust the labels on the pull
    request to track its state and progress towards merging.
+
 #. Once the PR has been reviewed and the CI tests have passed, the PR will be
    merged by one of the repository owners. In case this does not happen, ping
    us on Slack.
 
+#. If reviewers have requested for changes and once those changes are done,
+   re-request a review for the reviewers that have requested changes. Otherwise,
+   those reviewers will not be notified and your PR will not receive any
+   reviews.
+
+Pull requests review process for committers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Every committer in the `committers team <https://github.com/orgs/cilium/teams/committers/members>`_
+   belongs to at least to `one or more other teams in the Cilium organization <https://github.com/orgs/cilium/teams/team/teams>`_
+   if you would like to belong, or be removed from any team, please contact any
+   of the `maintainers <https://github.com/orgs/cilium/teams/maintainers/members>`_.
+
+#. Once a PR is open, the ``CODEOWNERS`` file will automatically pick which `teams <https://github.com/orgs/cilium/teams/team/teams>`_
+   should review the PR. Each committer can see which PR it needs to reviewed by
+   filtering which PRs it has been requested reviews for. A good filter is
+   provided in this `link <https://github.com/cilium/cilium/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse+review-requested%3A%40me+sort%3Aupdated-asc>`_
+   so make sure to bookmark it.
+
+#. Belonging to a team does not mean that a committer should know every single
+   line of code the team is maintaining, for this reason it is recommended
+   that, once a review has been given and if the committer does feel that it
+   would be better one other committer from his or her team would also review
+   the PR, the committer re-requests a review for the team that should review
+   the PR. In the example bellow, the committer belonging to the CI team is
+   re-requesting a review for other team members to review the PR. This allows
+   other team member belonging to the CI team to see the PR as part of the PRs
+   that require review in the `filter provided above <https://github.com/cilium/cilium/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse+review-requested%3A%40me+sort%3Aupdated-asc>`_
+
+   .. image:: ../../images/re-request-review.png
+      :align: center
+      :scale: 50%
+
+#. When all review objectives for all ``CODEOWNERS`` are met and all CI tests
```

Comment:
Maybe say "all required CI tests" here and put a link to https://docs.cilium.io/en/latest/contributing/testing/ci/#triggering-pull-request-builds-with-jenkins?

Reply:
I've added the "required". GH will automatically mark which jobs are required, that link is a little bit above in the section:
>``test-me-please`` as described in :ref:`trigger_phrases`. If you are a